### ISSUE TITLE
Gradients do not exist for variables

### ISSUE THREAD

**Issue by JkChenAcmer** [penny4860/tf-eager-yolo3#17]
When i'm training,it shows like this:

_initial_net_size
288 288 288
_initial_net_size
288 288 288
Training samples : 6537, Validation samples : 1044
Original yolov3 weights loaded!!

WARNING:tensorflow:Gradients do not exist for variables ['stage0/layer_0/moving_mean:0', 'stage0/layer_0/moving_variance:0',
...
, 'private__conv5_2/layer_103/moving_vari
ance:0', 'detection_layer_3_21/layer_104/moving_mean:0', 'detection_layer_3_21/layer_104/moving_variance:0'] when minimizing the loss.
  0%|                                                                                                                                     | 3/3268 [00:20<6:26:03,  7.09s/it]t

There are something wrong?

**Comment by penny4860** [comment]
Will the code in [2-training-from-scratch](https://github.com/penny4860/tf2-eager-yolo3#2-training-from-scratch) run properly?

**Comment by JkChenAcmer** [comment]
> Will the code in [2-training-from-scratch](https://github.com/penny4860/tf2-eager-yolo3#2-training-from-scratch) run properly

That‘s ok,despite the warning.

**Comment by JkChenAcmer** [comment]
> Will the code in [2-training-from-scratch](https://github.com/penny4860/tf2-eager-yolo3#2-training-from-scratch) run properly?
